bonding.py

```diff
@@ -1071,6 +1073,10 @@ def handle_args():
     peers_group.add_option('--nopeers', help='Do not run the peers portion of '
                                              'this utility',
                            action='store_true')
+    peers_group.add_option('--peerswait', help='The number of seconds to wait '
+                                               'for switch port negotiation. '
+                                               'Default [%default]',
```

Comment:
To make this more portable for switching between optparse and argparse, let's not use `%default`.  Since argparse changes this to `%(default)s` I would rather just see us use printf formatting, and duplicate the default by hardcoding it into the help string.